tests/README.md

```
@@ -1 +1,16 @@
-Automated tests live here.
\ No newline at end of file
+Automated tests live here.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)